src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -324,6 +324,8 @@ def k8s_install_cli(cmd, client_version='latest', install_location=None):
         _urlretrieve(file_url, install_location)
         os.chmod(install_location,
                  os.stat(install_location).st_mode | stat.S_IXUSR | stat.S_IXGRP | stat.S_IXOTH)
+        if os.path.isdir('/usr/share/bash-completion/completions/'):
```

Comment:
I don't think command logics should be coupled with ui completion only made for the docker image and for a specific shell.  I suggest using document/help to communicate